examples/csrApproval/csrApproval.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="JsonPatch.Net" Version="1.1.2" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\GenericKubernetesApi\GenericKubernetesApi.csproj" />
```

Comment:
what is this for?

Reply:
I added reference for using k8s client instead of downloading from nuget.

Reply:
see `Directory.Build.targets`
ref hiden there :)

you can remove it

Reply:
I hope now all should be good )